### ISSUE TITLE
[CS] Return music in main menu

### ISSUE THREAD

**Issue by XakepSDK** [ValveSoftware/halflife#2423]
Please return old music in main menu

**Comment by SamVanheer** [comment]
That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.

**Comment by RauliTop** [comment]
> That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.

Or just he can make a pull request to be available to everyone.
He didn't hurt anyone or anything.

**Comment by SamVanheer** [comment]
Pull requests are for SDK updates only (and only the code part of the SDK), which doesn't cover any version of CS.

I didn't imply that he was causing problems, i'm just saying it's something you can do yourself easily.

**Comment by sjain882** [comment]
> That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.

This is correct but you must understand that there is a difference between a mod and a feature

**Comment by SamVanheer** [comment]
> 
> 
> > That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.
> 
> This is correct but you must understand that there is a difference between a mod and a feature

What does that have to do with this issue?

**Comment by sjain882** [comment]
> > > That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.
> > 
> > 
> > This is correct but you must understand that there is a difference between a mod and a feature
> 
> What does that have to do with this issue?

Exactly what I mean for your original comment, I was trying to put it more nicely. Its completely irrelevant.

> That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.

We all know this is possible. What OP is asking (clear from the **title** of this issue) is to have it added back by default.

**Comment by SamVanheer** [comment]
> 
> 
> > > > That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.
> > > 
> > > 
> > > This is correct but you must understand that there is a difference between a mod and a feature
> > 
> > 
> > What does that have to do with this issue?
> 
> Exactly what I mean for your original comment, I was trying to put it more nicely. Its completely irrelevant.
> 
> > That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.
> 
> We all know this is possible. What OP is asking (clear from the **title** of this issue) is to have it added back by default.

If they start adding stuff back in people will start asking for pretty much every change to be undone. You can add the music back in yourself so i don't see the problem.

**Comment by sjain882** [comment]
> > > > > That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.
> > > > 
> > > > 
> > > > This is correct but you must understand that there is a difference between a mod and a feature
> > > 
> > > 
> > > What does that have to do with this issue?
> > 
> > 
> > Exactly what I mean for your original comment, I was trying to put it more nicely. Its completely irrelevant.
> > > That music was removed when CS was put on Steam more than 15 years ago. Condition Zero Deleted Scenes's menu music is apparently the same one so you can just copy it over to get it.
> > 
> > 
> > We all know this is possible. What OP is asking (clear from the **title** of this issue) is to have it added back by default.
> 
> If they start adding stuff back in people will start asking for pretty much every change to be undone. You can add the music back in yourself so i don't see the problem.

If that's your opinion, why not say it in the first place?

> If they start adding stuff back in people will start asking for pretty much every change to be undone

I don't see this happening for other similar issues.

**Comment by SamVanheer** [comment]
I was trying to be informative, i wasn't giving an opinion. I wasn't looking to argue either.

The point is that because you can do this yourself there is no reason for Valve to do it, since anyone who wants this can do it on their own and people who don't want it won't suddenly have music playing. This isn't like a code change where only Valve can do it, the engine automatically plays the file if it's there.

This request is based entirely on personal opinion, but the file had to be removed from the game installation by Valve at some point so they made the decision back then.

Providing the file in the game installation without making it play (automatically) would be a perfectly acceptable compromise, in fact adding all the soundtracks used in CS, CZ and DS would be a good idea.

**Comment by GBCS-PRO** [comment]
Hell yea I'd like to hear that officially back :)

**Comment by Lifeisstrange74** [comment]
The music was added first in a nosteam version everyone used.
It will never be readded, if you want menu music play CZ for Downed Pilot, or DS for the "classic 1.6" theme

**Comment by barfin** [comment]
I miss this so much it's shame that it's removed
